src/lizaalert/users/views.py

```diff
@@ -12,8 +12,21 @@
 
 
 class VolunteerAPIview(APIView):
+    """
+    Endpoint для работы с профилем пользователя.
+
+    Методы:
+    - GET: Отображение профиля пользователя.
+    - PATCH: Внесение изменений в профиль пользователя.
+
+    """
+
     permission_classes = [IsAuthenticated]
 
+    @swagger_auto_schema(
+        operation_description="Отображает профиль пользователя",
+        responses={200: VolunteerSerializer(), 204: "No Content", 404: "Not Found"},
```

Comment:
проверь и замени на актуальные значения

204 - возвращает ""
404 - json ?

```suggestion
        responses={200: VolunteerSerializer(), 204: {}, 404: "Not Found"},
```